omnibus.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency 'mixlib-versioning'
   gem.add_dependency 'ohai',             '~> 8.0'
   gem.add_dependency 'ruby-progressbar', '~> 1.7'
-  gem.add_dependency 'uber-s3'
+  gem.add_dependency 'uber-s3',          '~> 0.2.4'
```

Comment:
For pre-1.0, I think it's safer to only allow the patch version to float. SemVer allows breaking changes in any pre-1.0 release, but hopefully the authors would at least bump minor.